### DISCUSSION TITLE
Proxmox container script

### DISCUSSION THREAD

**Discussion by shot4free** [hargata/lubelog#583] [Show and tell]
Hello guys.

Thank you for such a great product. I enjoy it.

I automated lxc for LL here: https://github.com/shot4free/lubelog-proxmox-ct/tree/main

Feel free to use it or add it to your repository.

BR

**Comment by JcMinarro** [comment]
On [Community Script](https://github.com/community-scripts/ProxmoxVE) already [added](https://github.com/community-scripts/ProxmoxVE/pull/574) a way to install it directly on the LXC without using Docker.

**Reply by Sebastian-Voigt** [reply]
I have installed it and it works.
But how can I store or edit the .env file there?

**Reply by JcMinarro** [reply]
You don't really need this `.env` files.
This process install the services as Linux Standalone process.
You can follow the docs [here](https://docs.lubelogger.com/)

**Reply by Sebastian-Voigt** [reply]
Ok, but how can ich change the date format to dd/MM/yyyy and the currency to €?
According to the instructions you should set this in the .env.

**Reply by shot4free** [reply]
> Ok, but how can ich change the date format to dd/MM/yyyy and the currency to €? According to the instructions you should set this in the .env.

Please refer to line 40 of the script. Itt allows you to specify a custom .env file via a URL.
https://github.com/shot4free/lubelog-proxmox-ct/blob/5d503b21f9a529ce84b842e258c9ed923921cf16/setup-ll-ct.sh#L40

Alternatively, you can configure the environment using the LC_ALL variable as explained in the LubeLogger documentation.
https://docs.lubelogger.com/Advanced/Environment%20Variables

For future inquiries, please consider posting in the original repository where the script is maintained. The LubeLogger team is not responsible for this script, and their ability to assist is limited.

Hope this helps.
